typescript/baseDAO/src/generated/Set_quorum_threshold.ts

```diff
@@ -1,2 +1,5 @@
 import {Lambda} from '../common';
-export type Set_quorum_threshold = number;
+export interface Set_quorum_threshold {
```

Comment:
We should probably also update https://github.com/tqtezos/baseDAO/blob/pasqu4le/%23175-quorum-fraction/typescript/baseDAO/src/api.ts#L203

Most probably it ll have to be something like https://github.com/tqtezos/baseDAO/blob/pasqu4le/%23175-quorum-fraction/typescript/baseDAO/src/api.ts#L188

Reply:
@sras do you mean because of the argument?

If so I ended up finding more such cases and simplified those too.

Reply:
@pasqu4le Earlier the entrypoint argument was just a number, so this call could work

```
      contract => contract.methods.set_quorum_threshold(arg));
```

But now the entrypoint accept two values, so I think we ll have to pass the numerator and denominator separately as `contract.methods.set_quorum_threshold(arg.qtNumerator, arg.qtDenominator))`.

I am not sure though. If you have this contract deployed somewhere, I can check it.

Reply:
Oh I see, I have made the opposite change in the last commit instead.

I don't have a deployed contract to share, if you have the time please originate one and do check this.

Reply:
Checked with a deployed contract, and the current code gives the following error

```
InvalidParameterError {
  smartContractMethodName: 'set_quorum_threshold',
  sigs: [ [ 'nat', 'nat' ] ],
  args: [ { qtNumerator: 1, qtDenominator: 2 } ],
  name: 'Invalid parameters error',
  message: 'set_quorum_threshold Received 1 arguments while expecting one of the following signatures ([["nat","nat"]])'
}
```

I could get a successful call by modifying the api code for `set_quorum_threshold` as

```
  set_quorum_threshold(arg: Set_quorum_threshold): Promise<string|void> {
    return this.withContract(
      contract => contract.methods.set_quorum_threshold(arg.qtNumerator, arg.qtDenominator));
  }
```

Here is the deployed contract and storage after the call   ```BaseDAO.set_quorum_threshold({qtNumerator: 1, qtDenominator: 2});```


https://edo2net.tzkt.io/KT1HABxCoj7QYoErXa7HX6j3qquTPRMJU19V/storage

Reply:
@sras thanks a lot for checking.

I changed it now (and reverted the incorrect changes), I also modified the `HasAnnotation` instance of `QuorumThreshold` to not contain the prefix, so it's `numerator` and `denominator`.

These changes are all in the (only) fixup commits so you can check them.

Reply:
Np, LGTM now.